docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -609,13 +609,118 @@ Handy:
+    XP: 1
+    PP: [0, 1]
+    Save:
+        Trigger: Once
+        DR: 2
+        Type: GUT
+        Fail: Blinded
+    Range: 6
+    Prereq:
+        Role:
+            - Caster
+            - Support
+        Skill: Craft > 0
+
+Minor Levitation:
+    Type: Minor
+    Category:
+        - Magic
+    Description: Your skills allow you to momentarily suspend the laws of gravity.
+    Mechanic: You can infuse an object of less than 100 lbs with levitation, reducing its effective weight to 0 lbs and allowing it to float in the air. Additionally, you may expend 1 PP to force a target within 6 spaces to make a DR 2 STR Save or reduce their speed to 0 and float 1 space off the ground for 1 round.
+    XP: 1
+    PP: [0, 1]
+    Save:
+        Trigger: Once
+        DR: 2
+        Type: STR
+        Fail: Speed reduced to 0 and floating 1 space off the ground for 1 round.
+    Range: 6
+    Prereq:
+        Role:
+            - Caster
+            - Support
+        Skill:
+            - Craft > 0
+            - Knowledge > 0
+
+Produce Sound:
+    Type: Minor
+    Category:
+        - Magic
+    Description: Your skills let you create illusory sounds.
+    Mechanic: You can create a harmless sound or auditory effect emanating from a point you can see such as music, wind, rustling leaves, etc. for 1 minute. Additionally, you may expend 1 PP to attempt to feign the voice or noise of someone not there. A nearby creature must make a DR 2 Detection check to determine if the sound is illusory.
+    XP: 1
+    PP: [0, 1]
+    Range: 12
+    Prereq:
+        Role:
+            - Caster
+            - Support
+        Skill:
+            - Craft > 0
+            - Knowledge > 0
+
+Mystic Cleanse:
     Type: Minor
+    Category:
+        - Magic
+    Description: Your skills let you cleanse objects and creatures.
+    Mechanic: You can clean objects within 1 space, removing any dirt or grime. Damaged objects are not repaired, but dirt and debris around the damage is cleaned. Additionally, for 1 PP you can target a creature within 6 spaces and remove one of the following Status Conditions from them; Burned, Frozen, Poisoned, Blinded, Deafened, Charmed, Enthralled.
     XP: 1
+    PP: [0, 1]
+    Range: 6
+    Prereq:
+        Role:
+            - Caster
+            - Support
+        Skill:
+            - Craft > 0
+            - Knowledge > 0
+
+Illusory Appearance:
+    Type: Minor
+    Category:
+        - Magic
+    Description: Your skills let you conjure illusory effects on your person.
+    Mechanic: You can alter the appearance of your eye or hair color for 1 hour. Additionally, for 1 PP you can alter your appearance in a more meaningful way or produce an illusory aspect of your appearance for 1 hour. During this time, gain the Upper Hand on Conviction Checks made to convince others of your new appearance.
```

Comment:
```suggestion
    Mechanic: For 0 PP, you alter the appearance of you eye or hair color for 1 hour.  For 1 PP, you alter your appearance entirely, changing some or all physical characteristics. Occupying additional space (e.g., gaining height) produces illusions that may be readily detected if dramatically different from your original form. For the next hour, you gain Upper Hand on Conviction Checks made to convince others of your new appearance.
```

Reply:
Added this modified suggestion in latest commit.